x11-plugins/gkrellm-cpupower/files/gkrellm-cpupower-0.2-makefile.patch

```diff
@@ -0,0 +1,35 @@
+--- a/Makefile
++++ b/Makefile
+@@ -3,17 +3,22 @@
+ GTK_INCLUDE = `pkg-config gtk+-2.0 --cflags`
```

Comment:
While at it: replace `pkg-config` with `$(PKG_CONFIG)` or similar (defaulting to `pkg-config`) to respect user's choice of pkg-config.

While `pkg-config` should respect appropriate environment variables such that it's not needed to override it really, that doesn't mean much.

Reply:
I replaced it with:

```makefile
PKG_CONFIG = pkg-config # is this fine for the default?

GTK_INCLUDE = $(shell ${PKG_CONFIG} gtk+-2.0 --cflags)
GTK_LIB = $(shell ${PKG_CONFIG} gtk+-2.0 --libs)
```

Reply:
@thesamesam Should it be doing `tc-export PKG_CONFIG` as in the ebuild as well?

Reply:
yes!

Reply:
OK, then I will probably do:

```
BDEPEND="virtual/pkgconfig"
```
coupled with

```
tc-export PKG_CONFIG
```

in the ebuilds where this patch is added.

Reply:
I've gone through the already bumped ebuilds and added patches + BDEPEND for pkg-config.